src/planscape/forsys/parse_forsys_output.py

```diff
@@ -10,7 +10,7 @@ class RankedProject(TypedDict):
     id: int
     # Contribution of each priority to the total score
     # The contribution is weighted according to a scenario's priority weights.
-    weighted_priority_scores: dict
+    weighted_priority_scores: dict[str, float]
     # The total score, summed across weighted priority scores.
     total_score: float
```

Comment:
what's the semantic meaning of this/how is it relevant to the user's results? thinking ahead on how we explain these outputs

Reply:
I've updated the comments - I hope it's clearer.